app/lib/services/voip/CallNavRouter.test.ts

```
@@ -0,0 +1,191 @@
+	__esModule: true,
+	default: {
+		back: (...args: unknown[]) => mockBack(...args),
+		getCurrentRoute: (...args: unknown[]) => mockGetCurrentRoute(...args),
+		// Start with no navigation ref (not ready).
+		navigationRef: { current: null }
+	},
+	waitForNavigationReady: jest.fn().mockResolvedValue(undefined)
+}));
+
+// Import after mocks are set up.
+import { callLifecycle } from './CallLifecycle';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)